scripts/Component.py

```diff
@@ -86,7 +86,7 @@ def getFilters(self, mapping, config):
                     "Ice/plugin",
                     "Ice/stringConverter",
                     "Ice/threadPoolPriority",
-                    "Ice/udp",
```

Comment:
Not clear why we excluded udp test, seems to work fine. We no longer build service libraries for iOS.